keyform/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+from django.forms.widgets import RadioSelect, CheckboxSelectMultiple
+from django.forms import TypedChoiceField
+from keyform.models import Request
+
+class CreateForm(forms.ModelForm):
+    class Meta:
+        model = Request
+        fields = ['building', 'student_name', 'reason_for_request', 'amt_recieved', 'payment_method', 'charge_amount', 'staff', 'bpn', 'created_timestamp', 'charged_on_rcr']

```

Comment:
You should probably exclude 'created_timestamp' and set that automatically when the form is submitted.